src/main/webapp/style.css

```diff
@@ -1,18 +1,24 @@
 body {
   background-color: #ececec;
   color: black;
-  font-family: Aksara Bali Galang;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
I just want to check here, since a lot of the changes you make in here will affect the whole site - did the site's design change, or do you just need to use the defaults based on the current design?

Reply:
Ah, I just saw Sydney's changes. In that case, I assume most of them are a redesign across the site.

Reply:
That's because linter warns that the previous font-family is not found/supported :)

Reply:
Oh, if I remember the lint warning correctly, what they really wanted was one of the fonts to be generic, so that there was a fallback. You can still use custom fonts if you have them downloaded from Google Fonts or something similar. But you need to end with something like `sans-serif`.